tests/e2e/ansible/install_build_deps.yml

```diff
@@ -32,6 +32,19 @@
         dest: /usr/local/bin/operator-sdk
         mode: '+x'
     - import_tasks: "install_docker.yml"
+      # Docker buildx relies on qemu-user-static to multi-arch builds, but
+      # qemu-user-static is not packaged for CentOS. Let's get it installed via
+      # https://github.com/multiarch/qemu-user-static
+    - name: Handle qemu-user-static installation on CentOS.
+      block:
+        - name: Check qemu-user-static is installed
+          shell: docker run --rm -t s390x/ubuntu uname -m
```

Comment:
The qemu-user-static is used for buildx to build for build architectures. This line in particular I am checking if the qemu-user-static is already installed (i.e. if I can run a s390x container on x86_64) otherwise I try to install it on the node with `docker run --rm --privileged multiarch/qemu-user-static --reset -p yes`.

In a VM with CentOS Stream 8 that logic works. I suspect that on the TDX machine it doesn't because, well, the images are on docker hub...

Could you please try to manually install on the machine with:
```
sudo docker run --rm --privileged multiarch/qemu-user-static:7.2.0-1 --reset -p yes
```

I will change the scripts to use `multiarch/qemu-user-static:7.2.0-1` instead of `latest` so hopefully it will use the cached image.

Reply:
in TDX CI server:
```
docker run --rm --privileged multiarch/qemu-user-static:7.2.0-1 --reset -p yes
Unable to find image 'multiarch/qemu-user-static:7.2.0-1' locally
7.2.0-1: Pulling from multiarch/qemu-user-static
Digest: sha256:fe60359c92e86a43cc87b3d906006245f77bfc0565676b80004cc666e4feb9f0
Status: Downloaded newer image for multiarch/qemu-user-static:7.2.0-1
Setting /usr/bin/qemu-alpha-static as binfmt interpreter for alpha
Setting /usr/bin/qemu-arm-static as binfmt interpreter for arm
Setting /usr/bin/qemu-armeb-static as binfmt interpreter for armeb
Setting /usr/bin/qemu-sparc-static as binfmt interpreter for sparc
Setting /usr/bin/qemu-sparc32plus-static as binfmt interpreter for sparc32plus
Setting /usr/bin/qemu-sparc64-static as binfmt interpreter for sparc64
Setting /usr/bin/qemu-ppc-static as binfmt interpreter for ppc
Setting /usr/bin/qemu-ppc64-static as binfmt interpreter for ppc64
Setting /usr/bin/qemu-ppc64le-static as binfmt interpreter for ppc64le
Setting /usr/bin/qemu-m68k-static as binfmt interpreter for m68k
Setting /usr/bin/qemu-mips-static as binfmt interpreter for mips
Setting /usr/bin/qemu-mipsel-static as binfmt interpreter for mipsel
Setting /usr/bin/qemu-mipsn32-static as binfmt interpreter for mipsn32
Setting /usr/bin/qemu-mipsn32el-static as binfmt interpreter for mipsn32el
Setting /usr/bin/qemu-mips64-static as binfmt interpreter for mips64
Setting /usr/bin/qemu-mips64el-static as binfmt interpreter for mips64el
Setting /usr/bin/qemu-sh4-static as binfmt interpreter for sh4
Setting /usr/bin/qemu-sh4eb-static as binfmt interpreter for sh4eb
Setting /usr/bin/qemu-s390x-static as binfmt interpreter for s390x
Setting /usr/bin/qemu-aarch64-static as binfmt interpreter for aarch64
Setting /usr/bin/qemu-aarch64_be-static as binfmt interpreter for aarch64_be
Setting /usr/bin/qemu-hppa-static as binfmt interpreter for hppa
Setting /usr/bin/qemu-riscv32-static as binfmt interpreter for riscv32
Setting /usr/bin/qemu-riscv64-static as binfmt interpreter for riscv64
Setting /usr/bin/qemu-xtensa-static as binfmt interpreter for xtensa
Setting /usr/bin/qemu-xtensaeb-static as binfmt interpreter for xtensaeb
Setting /usr/bin/qemu-microblaze-static as binfmt interpreter for microblaze
Setting /usr/bin/qemu-microblazeel-static as binfmt interpreter for microblazeel
Setting /usr/bin/qemu-or1k-static as binfmt interpreter for or1k
Setting /usr/bin/qemu-hexagon-static as binfmt interpreter for hexagon
```

```
docker run --rm -t s390x/ubuntu:22.04 uname -m
Unable to find image 's390x/ubuntu:22.04' locally
22.04: Pulling from s390x/ubuntu
ccff9014dcde: Pull complete
Digest: sha256:107cc410d683e41f2765125ec3b3aba514025348590539fcc079fa02d54f5708
Status: Downloaded newer image for s390x/ubuntu:22.04
WARNING: The requested image's platform (linux/s390x) does not match the detected host platform (linux/amd64/v4) and no specific platform was requested
```